app/importers/file_importers/educator_section_assignments_importer.rb

```diff
@@ -32,7 +32,10 @@ def filter
 
   def delete_rows
     #Delete all stale rows no longer included in the import
-    EducatorSectionAssignment.where.not(id: @imported_assignments).delete_all
+    #For the schools imported during this run of the importer
+    EducatorSectionAssignment.joins(:section => {:course => :school})
+                             .where.not(id: @imported_assignments)
+                             .where(:schools => {:local_id => school_scope}).delete_all
```

Comment:
If I read right, this is expecting caps for `local_id` (versus lowercase for the slug).  Any downside to upcasing here to catch mistakes like `heroku run:detached thor import:start --school shs` etc.?

Reply:
If we did upcase, I think it would make sense in the import.thor since a lowercase school would affect all the other importers as well.

Reply:
👍 yeah good idea, seems like a more logical spot to put it as well.  The name is `school_scope` in importers so it's ambiguous whether that's supposed to be a slug or local_id and you have to trace the execution to tell (no need to change that now).